CourseStream/CourseStream.Data/Migrations/20231121172427_EnrolledStudentsTableAdded.Designer.cs

```diff
@@ -0,0 +1,78 @@
+﻿// <auto-generated />
+using CourseStream.Data;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Infrastructure;
+using Microsoft.EntityFrameworkCore.Migrations;
+using Microsoft.EntityFrameworkCore.Storage.ValueConversion;
+using Npgsql.EntityFrameworkCore.PostgreSQL.Metadata;
+
+#nullable disable
+
+namespace CourseStream.Data.Migrations
+{
+    [DbContext(typeof(CourseStreamDBContext))]
+    [Migration("20231121172427_EnrolledStudentsTableAdded")]
+    partial class EnrolledStudentsTableAdded
+    {
+        /// <inheritdoc />
+        protected override void BuildTargetModel(ModelBuilder modelBuilder)
+        {
+#pragma warning disable 612, 618
+            modelBuilder
+                .HasAnnotation("ProductVersion", "7.0.13")
+                .HasAnnotation("Relational:MaxIdentifierLength", 63);
+
+            NpgsqlModelBuilderExtensions.UseIdentityByDefaultColumns(modelBuilder);
+
+            modelBuilder.Entity("CourseStream.Data.CourseStream", b =>
+                {
+                    b.Property<int>("Id")
+                        .ValueGeneratedOnAdd()
+                        .HasColumnType("integer");
+
+                    NpgsqlPropertyBuilderExtensions.UseIdentityByDefaultColumn(b.Property<int>("Id"));
+
+                    b.Property<string>("CourseName")
+                        .IsRequired()
+                        .HasColumnType("text");
+
+                    b.HasKey("Id");
+
+                    b.ToTable("CourseStreams");
+                });
+
+            modelBuilder.Entity("CourseStream.Data.EnrolledStudent", b =>
+                {
+                    b.Property<int>("Id")
+                        .ValueGeneratedOnAdd()
+                        .HasColumnType("integer");
+
+                    NpgsqlPropertyBuilderExtensions.UseIdentityByDefaultColumn(b.Property<int>("Id"));
+
+                    b.Property<int>("CourseStreamId")
+                        .HasColumnType("integer");
+
+                    b.Property<int>("StudentId")
+                        .HasColumnType("integer");
+
+                    b.HasKey("Id");
+
+                    b.HasIndex("CourseStreamId");
+
+                    b.ToTable("EnrolledStudents");
+                });
+
+            modelBuilder.Entity("CourseStream.Data.EnrolledStudent", b =>
```

Comment:
Probably it makes sense to rename this to `Enrollment`. Not a requirement, just a nit

Reply:
Renamed to "Enrollment"